clusters/hive/hypershift/README.md

```diff
@@ -0,0 +1,21 @@
+This directory includes the files (include generated files) for deploying HyperShift on `hive`.
+
+# HyperShift CLI and operator images
+HyperShift CLI (the utility for installing HyperShift, as well as creting/destroying hosted clusters) and HyperShift operator (the core component on HyperShift management cluster) share the same image, due to the [fact](https://issues.redhat.com/browse/HOSTEDCP-792) that HyperShift does not provide backward or forward compatibilities.
+
+The image is built with `clusters/app.ci/supplemental-ci-images/hypershift/hypershift-cli.yaml` to available for all build farms.
+
+# Deployment/upgrading on `hive`
+Prepare the AWS credentials for your management cluster account in `~/workspace/mgmt-aws` (or any other path you prefer).
```

Comment:
- Did you use personal credentials? I'd like to know what to do if I need to redo hypershift.
- Could we run the cmd without `~/workspace/mgmt-aws`? Easy for copy/paste.

Reply:
> Did you use personal credentials? I'd like to know what to do if I need to redo hypershift.
the `mgmt-aws` credential is the credential for the AWS account for the management cluster. In this case, its a credential to the AWS account hosting `hive`. Its one time used only (during the installation).

> Could we run the cmd without ~/workspace/mgmt-aws? Easy for copy/paste.
yes, but we still need to specify the files...

Reply:
The credentials to which AM user? Under which AWS account?
Let us clarify it in the doc if it does not expose sensitive information.

>yes, but we still need to specify the files...

I do not understand this.